test/typing/plain_files/orm/relationship.py

```diff
@@ -68,12 +65,12 @@ class Address(Base):
     email_name: Mapped[str] = mapped_column("email_name")
 
     user_style_one: Mapped[User] = relationship()
-    user_style_two: Mapped["User"] = relationship()
```

Comment:
this is explicitly testing that both`Mapped[User]` and `Mapped["User"]` are accepted